site/docs/tooltip.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+menu: docs
+title: Tooltip
+permalink: /docs/tooltip.html
+---
+
+Tooltip can provide details-on-demand about various marks on Vega-lite visualization. There are two distinct ways to create a tooltip
+
+* TOC
+{:toc}
+
+## Using Tooltip channel
+
+Vega-lite provided [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel to create tooltip using SVG [`title`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) tag. For example, this bar chart supports tooltip for field `b`. Hover over the bar will show a simple tooltip that displays individual value of field `b` of each bar.
+
+<div class="vl-example" data-name="bar_tooltip"></div>
+
+You can see that [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel provides a very limited functionality for tooltip since only one field can be displayed at a time. Moreover, no customization of the tooltip can be done using [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel.
+
+## Using Vega-tooltip plugin
+
+[`Vega-tooltip`](https://github.com/vega/vega-tooltip/) is a tooltip plugin for both Vega and Vega-lite visualization. It generates tooltip by creating HTML element, rather than using SVG title that is produced by using Marks channel. However, [`Vega-tooltip`](https://github.com/vega/vega-tooltip/) provides more functionality than using [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel. It allows you to specify fields and values to display, as well as the customization of the tooltip.
```

Comment:
> It generates tooltip by creating HTML element, rather than using SVG title that is produced by using Marks channel.   However, [`Vega-tooltip`](https://github.com/vega/vega-tooltip/) provides more functionality than using [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel. 

The latter sentence doesn't really contradict the first.  Maybe:

"By using HTML elements to generate tooltip, Vega-tooltip provides more features compared to the built-in `tooltip` channel.    (Please add link back)"

> It allows you to specify fields and values to display

I think the key part is "It can display a table showing multiple fields in the tooltip".  

> , as well as the customization of the tooltip.

Since the built in support customization too, maybe more specific about what kind of customization is possible?